### PR TITLE
Added contribution for linear operators

### PR DESCRIPTION
**Pull request by jppelteret**
@tamiko Ping!

### CODE REVIEW

**Comment by tamiko** [comment]
@jppelteret Will hijack and merge later today :-) Thanks a lot!

File: 8.5/paper.tex [hunk 1]
```diff
@@ -177,9 +176,16 @@ \subsection{The \texttt{MappingManifold} class}
...
 
-\subsection{TODO: Linear operators with Payload and Trilinos}
+\subsection{Extension of the \texttt{LinearOperator} class}
 
-TODO: Jean-Paul and Matthias
+By introduction of an arbitrary \texttt{Payload} base class, the \texttt{LinearOperator} class can be extended to seamlessly exploit the native features and operations offered by external linear algebra libraries.
+We have thus developed a \texttt{TrilinosPayload} class, through which full support for the suite of {\trilinos} parallel iterative solvers and preconditioners has been achieved.
+In the case of the construction of an \texttt{inverse\_operator}, both \dealii's built in solvers and those offered by {\trilinos} can be selected.
+This has been achieved by using the \texttt{Epetra\_Operator} as the basis for the \texttt{TrilinosPayload}, for which the result of both standard and composite operations involving forward (\texttt{Apply()}) and inverse (\texttt{ApplyInverse()}) matrix-vector multiplication are collated using lambda functions.
+It is envisaged that, in the future, similar can be done for the {\petsc} iterative solvers.
```
Comment (bangerth) [inline 8.5/paper.tex]: -> We envisage

Let's try to avoid passive voice.

File: 8.5/paper.tex [hunk 2]
```diff
@@ -177,9 +176,16 @@ \subsection{The \texttt{MappingManifold} class}
...
+
+An additional feature of the \texttt{LinearOperator} suite is the defintion of a \texttt{schur\_complement} operator, and its associated condensation and post-processing \texttt{PackagedOperation}s.
+An operator representing the Schur complement of a block system can be declared and, though the delayed evaluation offered by \texttt{PackagedOperation}s, reused on any number of vector systems.
```
Comment (bangerth) [inline 8.5/paper.tex]: though -> through?

**Comment by bangerth** [comment]
I'm merging this -- let's not go through pull requests but instead just commit what we have and edit each others' texts.

**Comment by jppelteret** [comment]
@bangerth By private mail, @tamiko and I had agreed to work on this together --> PR

**Comment by bangerth** [comment]
Well, I will let you two figure it out. But I have no problem if you just take turns editing the same document and pushing your changes into the repo right away. There is no testsuite you could break if others don't agree with your text, and it's so much easier to just edit someone's text than to annotate it on github.

But do as the two of you feel most comfortable collaborating :-)

**Pull request merged**